src/Steps/PermissionCheckStep.spec.ts

```diff
@@ -0,0 +1,84 @@
+import PermissionCheckStep from './PermissionCheckStep';
+
+import RolesStore from '../RolesStore';
+
+import AccessControlError from '../Errors/AccessControlError';
+
+import rolesFixture from '../__fixtures__/roles';
+
+import { GrantQuery } from '../types';
+
+describe('PermissionCheckStep', () => {
+  it('throw error if `rolesStore` is not setup when calling `hasPermission`', () => {
```

Comment:
Test cases should read like `it('should return false when permission is denied')`.